client/test/preview/integration/route/digitaltwins/execute/PipelineUtils.test.tsx

```diff
@@ -1,38 +1,41 @@
-import { JobSchema } from '@gitbeaker/rest';
 import * as PipelineUtils from 'preview/route/digitaltwins/execute/pipelineUtils';
+import cleanLog from 'model/backend/gitlab/cleanLog';
 import { setDigitalTwin } from 'preview/store/digitalTwin.slice';
-import { mockDigitalTwin } from 'test/preview/__mocks__/global_mocks';
+import { mockGitlabInstance } from 'test/preview/__mocks__/global_mocks';
 import { previewStore as store } from 'test/preview/integration/integration.testUtil';
+import { JobSchema } from '@gitbeaker/rest';
+import DigitalTwin from 'preview/util/digitalTwin';
 
 describe('PipelineUtils', () => {
-  const digitalTwin = mockDigitalTwin;
-  digitalTwin.lastExecutionStatus = 'success';
-
-  const { gitlabInstance } = digitalTwin;
-
+  let digitalTwin: DigitalTwin;
+  
   beforeEach(() => {
+    digitalTwin = new DigitalTwin('mockedDTName', mockGitlabInstance);
     store.dispatch(setDigitalTwin({ assetName: 'mockedDTName', digitalTwin }));
+    
+    digitalTwin.execute = jest.fn().mockImplementation(async () => {
+      digitalTwin.lastExecutionStatus = 'success';
+      return Promise.resolve();
+    });
   });
-
+  
   afterEach(() => {
+    jest.restoreAllMocks(); 
     jest.clearAllMocks();
   });
-
+  
```

Comment:
The extra space is not a requirement. Please restore the changes to the empty lines.